extensions/podman/packages/extension/package.json

```diff
@@ -144,10 +144,10 @@
           "when": "podman.podmanMachineDiskSupported"
         },
         "podman.factory.machine.image-path": {
+          "markdownDescription": "Image path (optional) in a form of fully qualified registry, path, or URL to a VM image. Registry target must be in the form of _docker://registry/repo/image:version_.",
           "type": "string",
-          "format": "file",
-          "scope": "ContainerProviderConnectionFactory",
-          "description": "Image Path (Optional)"
+          "format": "file,url,fullyQualifiedRegistry",
```

Comment:
`--image-path` option accepts three different formats and this change is reflecting that. Other approaches would increase complexity and require more changes in codebase.

Reply:
it's definitely more complex to introduce more format options

I could use several exclusive formatter for a given type that could be worst

Why in the UI we can't have a field to specify a file path and another field to specify an URI

Then if I select both options, as we can validate parameters it will say that it is incorrect and if I select a file I have the nice browse and select window

Under the hood it can call the same parameter to give image-path

Reply:
> Then if I select both options, as we can validate parameters it will say that it is incorrect and if I select a file I have the nice browse and select window

BTW, you still have nice browse and select window in this PR at 2:05.